contracts/libraries/OracleLibrary.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity =0.7.6;
+
+import '@uniswap/v3-core/contracts/libraries/FullMath.sol';
+import '@uniswap/v3-core/contracts/libraries/TickMath.sol';
+import '@uniswap/v3-core/contracts/interfaces/pool/IUniswapV3PoolDerivedState.sol';
+import '@uniswap/v3-core/contracts/libraries/LowGasSafeMath.sol';
+import 'hardhat/console.sol';
+
+library OracleLibrary {
+    function consult(
+        uint32 time,
```

Comment:
we are pretty bad about this in core, but this should probably be `blockTimestampCurrent` instead of just `time`